dags/openshift_nightlies/scripts/install/rosa.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -ex
+
+while getopts v:a:j:o: flag
+do
+    case "${flag}" in
+        v) version=${OPTARG};;
+        a) version_alias=${OPTARG};;
+        j) json_file=${OPTARG};;
+        o) operation=${OPTARG};;
+        *) echo "ERROR: invalid parameter ${flag}" ;;
+    esac
+done
+
+
+setup(){
+    mkdir /home/airflow/workspace
+    cd /home/airflow/workspace
+    export PATH=$PATH:/usr/bin
+    export AWS_REGION=us-west-2
+    export AWS_ACCESS_KEY_ID=$(cat ${json_file} | jq -r .aws_access_key_id)
+    export AWS_SECRET_ACCESS_KEY=$(cat ${json_file} | jq -r .aws_secret_access_key)
+    export ROSA_TOKEN=$(cat ${json_file} | jq -r .rosa_creds_token)
+    export CLUSTER_NAME=$(cat ${json_file} | jq -r .openshift_cluster_name)
+    rosa login --env=staging
+}
+
+install(){
+    rosa whoami
+    rosa verify quota
+    rosa verify permissions
```

Comment:
I assume the above rosa commands are to verify things but shouldn't we be exiting on failure?

Reply:
if any of them fails, script ends with exit code 1, there is no need to get result.

actually, im going to remove the exit code verification on the rosa create cluster, because it is not being used, thats from an old version with "|| true" after the command

Reply:
in that case should we move the verification to the setup section after the login?

Reply:
moved